wandb/integration/openai/__init__.py

```diff
@@ -10,7 +10,11 @@
 from .openai import autolog
 
 if _openai:
-    if parse_version(_openai.__version__) > parse_version("0.28.1"):
+    if parse_version(_openai.__version__) > parse_version("1.0.1"):
         from .fine_tune import WandbLogger
 else:
-    wandb.termerror("`openai` is not installed. To fix, please `pip install openai`")
+    wandb.termerror(
```

Comment:
is this correct for autolog? or only for fine_tune (the version)

Reply:
I have heard deprecating autolog in some conversation - I am not sure though. Also autolog will work with <=0.28.1.

Reply:
ok so let's remove all this version check from init and move it to fine_tune

Reply:
leave the init file like it was before this PR

Reply:
The version check in the init is important for the tests. Other openai functional tests fail because they install v0.28.1 and WandbLogger fails to be imported.